docs/deployment/uwsgi_cookiecutter_2__emperor.rst

```diff
@@ -65,9 +73,10 @@ WSGI entry point.
 
         [uwsgi]
         ...
-        wsgi-file = wsgi.py                 # Used in part 2 of this tutorial
-        callable = app                      # Used in part 2 of this tutorial
-        logto = /var/log/uwsgi/%(proj).log  # Used in part 2 of this tutorial
+        # Uncomment `wsgi-file`, `callable`, and `logto` during Part 2 of this tutorial
+        wsgi-file = wsgi.py
+        callable = app
+        logto = /var/log/uwsgi/%(proj).log
```

Comment:
Changed the comment scheme here, as `#` works at the beginning of a line for a comment, but does not work partway through a line.